api/v1/paths/developers.js

```diff
@@ -1,22 +1,22 @@
 const appRoot = require('app-root-path');
 
-const petsDao = require('../db/json/pets-dao-example');
+const developersDao = require('../db/oracledb/developers-dao');
```

Comment:
Since you only need the `getDevelopers` method from `developersDao`, it might be simpler to destructure just that method

Reply:
I will need it for POST in the future